### PR TITLE
New option for Waveshare 2.13 (D) display

### PR DESCRIPTION
**Pull request by lightisfaster**
Based on the implementation of waveshare2in13, the unsupported commands FULL_UPDATE and PART_UPDATE have been eliminated. Furthermore, the alignment of the messages has been adjusted.

### CODE REVIEW

**Comment by k9ert** [comment]
What's missing here for this to get merged?

**Comment by lightisfaster** [comment]
Some texts on the screens were not finally positioned. This is done, now.

File: config.py
Comment (k9ert) [inline config.py:138]: These options are mutually exclusive. I'm not sure how to install the library (maybe i did that wrong) but why not using `elif` constructs? This fixed my issue.

Comment (lightisfaster) [reply config.py:138]: That's correct. I took the normal if so as not to touch the papirus block. If the correct display is choose in the config.ini, there are no problems either.

Comment (lightisfaster) [reply config.py:138]: We can make the change with a separate PR

Comment (k9ert) [reply config.py:138]: I'm not sure i understand the approach. Why don't you want to touch the papirus block? This is a configuration procedure and there is some kind of fallback-mode via "waveshare" (as it's not 100% specified). However, that works with elif as well and these values are mutually exclusive. Using "if", not elif, makes it error-prone: I had issues with the normal if-block. Maybe i missed the point somewhere: Where is the description of the installation of the waveshare-library? Maybe i did that wrong.
But that should not prevent doing elifs?!

**Comment by k9ert** [comment]
Ok, so yesterday i could prove that it's working in "hat-mode". When i started the `app.py` it resulted in "LightningATM" Logo and "insert coins". However, when i started to wire as described in the video, it stopped working. I rewired twice and triple-checked each time but couldn't get it to work.
I'm not sure where this is coming from. Maybe one of my wires have a defect. It's very unlikely to have anything to do with this PR but wanted to mention it nevertheless.

After solving the elif-issue, i think this is ready to merge, but what do i know.

File: config.py
Comment (lightisfaster) [reply config.py:138]: If you look at the existing PRs, you will see the touch of the papirus block will results in additional merge conflicts. I would therefore prefer to have a new PR with the elif. Then we can choose the time when we approve the adjustment. I have no problems using the normal if.

**Comment by lightisfaster** [comment]
It's really strange.
Unfortunately I can't comment on that.
@21isenough has write access and can edit the code before merge. If he think it is a good idea to use elif, he can do this. Otherwise we will apply for the elif change afterwards.

**Review by 21isenough — APPROVED** [review]

**Pull request merged**
